src/ApiGateway/SimpleWebsocketClient.php

```diff
@@ -0,0 +1,62 @@
+<?php declare(strict_types=1);
+
+namespace Bref\ApiGateway;
+
+use AsyncAws\Core\Request;
+use AsyncAws\Core\Stream\StreamFactory;
+use Symfony\Component\HttpClient\HttpClient;
+
+/**
+ * A simple alternative for communicating with websocket connections.
+ */
+final class SimpleWebsocketClient
+{
+    /** @var string */
+    private $stage;
+
+    /** @var WebsocketClient */
+    private $client;
+
+    public function __construct(string $apiId, string $region, string $stage, int $timeout = 60)
```

Comment:
```suggestion
    public function __construct(string $apiId, string $region, string $stage, int $timeout = 10)
```

Default timeouts at 60s are usually problematic because the timeout for HTTP requests is 30 seconds on Lambda. That means that we never know where the code was actually hanged. Because of that, and also because having users wait 60 seconds when a service is down is a bit much, I always advocate for shorter timeouts.